build/batchcopy.cmd

```diff
@@ -1 +1,14 @@
-@powershell -NoProfile -ExecutionPolicy unrestricted -File "%~dp0\batchcopy.ps1" %*
+@echo off
+
+set SRC=%1
+set DST=%2
+
+set SRC=%SRC:/=\%
+set DST=%DST:/=\%
+
+if not exist %SRC% goto end
```

Comment:
I notice the sh script creates the directory if it doesn't exists and carries on. Is there a reason for having different behavior between the two batchcopy scripts?